infrastructure/operators.yml

```diff
@@ -0,0 +1,32 @@
+---
+- hosts: default
```

Comment:
Could we link back to the host alias documentation in the packer ansible provisioner?